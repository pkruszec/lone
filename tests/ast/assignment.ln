// Constants (type inferred)
a := 5;
b := "test";
c := 's';
