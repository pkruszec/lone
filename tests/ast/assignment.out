PROG
  CONST
    IDENT(a)
    NUM(5)
  CONST
    IDENT(b)
    STR(test)
  CONST
    IDENT(c)
    CHAR(s)
